src/common/NoSSR/index.tsx

```diff
@@ -0,0 +1,20 @@
+import { ReactNode, useEffect, useState } from 'react';
+
+interface Props {
+  children: ReactNode,
+}
+
+const NoSSR = ({ children }: Props) => {
```

Comment:
Tuhle komponenty bych napsal jinak. Já mám context, ze kterého jde vyčíst, jestli jsi na FE nebo BE. Upravím to pak u sebe.

Reply:
Vyřešeno tím, že jsi jí smazal. :)